tests/testthat/test_align_plots.R

```diff
@@ -45,7 +45,8 @@ test_that("complex alignments, h, v, hv", {
   expect_equal(plots[[1]]$widths[1:4], plots[[2]]$widths[1:4])
 
   plots <- align_plots(p1, p2, align = "v", axis = "r", greedy = FALSE)  # align right
-  expect_equal(plots[[1]]$widths[6:9], plots[[2]]$widths[14:17])
+  ncol <- vapply(plots, ncol, integer(1))
```

Comment:
I know it's just a test but I don't really like having a variable (`ncol`) that looks the same as a built-in function (`ncol`). Could you change the variable name? `n` would be fine, as would be `nc`, or anything else really.